Makefile

```diff
@@ -54,6 +57,10 @@ docker: build
 	chmod 755 backend/backend
 	docker build backend -t opensdsio/multi-cloud-backend:latest
 
+	cp $(BUILD_DIR)/file file
+	chmod 755 file/file
+	docker build file -t opensdsio/multi-cloud-file:latest
+
```

Comment:
File may also need to be cleaned, onle line 81.

Reply:
okay. Done.
Thanks